build.xml

```diff
@@ -175,6 +177,10 @@
 		<property name="rhino.zip" value="rhino-${rhino.version}.zip"/>
 		<property name="rhino.zip.url" value="https://github.com/mozilla/rhino/releases/download/Rhino${rhino.version.major}_${rhino.version.minor}_${rhino.version.patch}_Release/${rhino.zip}"/>
 		<property name="rhino.dir" value="build/rhino${rhino.version}"/>
+		<property name="antform.base" value="antform-src-2.0"/>
+		<property name="antform.zip" value="${antform.base}.zip"/>
+		<property name="antform.zip.url" value="https://sourceforge.net/projects/antforms/files/antforms/AntForm%%202.0/${antform.zip}/download"/>
```

Comment:
Ik zie geen antform.gz.url property. Gaat dit werken onder Linux en Mac?

Reply:
Bestaat er een .gz url? Als hij te vinden is dan stel ik voor dat je hem gebruikt. Dan weten we zeker dat hij onder Linux goed wordt uitgepakt. Ik weet niet of de Frank!Runner er nu op berekend is om .zip files te gebruiken onder Linux. Als het mogelijk is zou ik het consistent willen hebben voor alle libraries.